autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>Not sure if you want to apply a different typeset to <code>Amount</code>  and other column references?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>